### DISCUSSION TITLE
What's the goal of `uniorg-rehype`?

### DISCUSSION THREAD

**Discussion by venikx** [rasendubi/uniorg#93] [Q&A] [answered]
Is the goal providing good defaults "a la" org-html, but having the capability to customise this?
I'm asking, cause I'm curious what to do with source blocks which have `:exports none`. In regular org-export, it means these code blocks aren't getting exported and should be ignored.

So that'd mean the following code should return null.
https://github.com/rasendubi/uniorg/blob/3ee16b8a6feb26b569293245fdbca649507067f5/packages/uniorg-rehype/src/org-to-hast.ts#L304-L317

Or is the goal of `uniorg-rehype` to provide the least destructive changes to generate the html, while proving decent defaults?

**Comment by rasendubi** [comment, marked as answer]
org-rehype mimicks ox-html export, so :export none should return null. It
might also need to look ahead and remove a result block as well

**Reply by venikx** [reply]
Gotcha, I’ll keep that in mind and probably will look into this myself as well👍🏼
